SynCrtSock.pas

```diff
@@ -6209,7 +6228,10 @@ constructor THttpServerResp.Create(aServerSock: THttpServerSocket; aServer: THtt
   finally
     LeaveCriticalSection(fServer.fProcessCS);
   end;
-  fConnectionID := fServer.NextConnectionID;
+  if (aServerSock.RemoteConnectionID > 0) then

```

Comment:
I think should be:
`aServerSock.RemoteConnectionID` <> 0

Reply:
+1